Tests/Unit/Controller/Admin/ModuleSupportTest.php

```diff
@@ -84,4 +84,66 @@ public function sendSupportEmailActionWithParamsProvider()
             'no body failure' => [null, 'from@email.com', 'reply@email.com', Helper::translate('wd_message_empty_error')],
         ];
     }
+
+    public function testAddDataFromForm()
+    {
+        $_POST['module_support_text'] = 'abcd';
+        $_POST['module_support_email_from'] = 'test@test.com';
+        $_POST['module_support_email_reply'] = 'test@test.com';
```

Comment:
Please, let's use **two distinct email strings** to be sure we are checking for the right one.

Reply:
Done!